### ISSUE TITLE
Parameter 'zone' should not be mandatory deploying CVO-HA on GCP

### ISSUE THREAD

**Issue by pablogarciaarevalo** [NetApp/terraform-provider-netapp-cloudmanager#146]
Deploying a CVO-HA on Google Cloud requires the parameter 'zone'.

_Error: Missing required argument
The argument "zone" is required, but no definition was found._

Using '**is_ha = true**' to specify that it's a HA deployment and not a Single Node, the 'zone' parameter should not be mandatory but the parameters 'node1_zone', 'node2_zone', and 'mediator_zone'.

**Comment by chuyich** [comment]
In here, the **zone** is actually 'GCP Region' on the UI
<img width="694" alt="image" src="https://user-images.githubusercontent.com/60237821/210632761-a8417491-a60c-437e-8262-bd89dbbff27e.png">

**Comment by pablogarciaarevalo** [comment]
This is not the meaning of a zone. Neither in the provider documentation:
_[zone](https://registry.terraform.io/providers/NetApp/netapp-cloudmanager/latest/docs#zone) - (Required) The zone of the region where the working environment will be created_

**Comment by chuyich** [comment]
The "zone" will be converted to "region". For example, if the zone is set as "us-east1-b", it will be converted to "us-east1" as "region" for creating the GCP HA CVO. Will update the document to clarify this.

**Comment by pablogarciaarevalo** [comment]
Thank you for quick your response. 

Regardless that IMHO deploying a CVO HA specifying a GCP region in the zone parameter, the same zone parameter that we use to specify the GCP zone deploying a CVO SN and we don't have to set the GCP region in other parameter is confused and inconsistent, I've just tested the HA deployment using a GCP region in the zone parameter and it fails with an unknown error.

```
Error: code: 400, message: {"message":"Bad Request","causeMessage":"SimplicatorBadRequestException: Bad Request"}
  with netapp-cloudmanager_cvo_gcp.cvo-gcp,
  on cvo_11.tf line 15, in resource "netapp-cloudmanager_cvo_gcp" "cvo-gcp":
  15: resource "netapp-cloudmanager_cvo_gcp" "cvo-gcp" {
```

Our CVO HA deployment example suggests indeed to use a GCP zone, not a region:
https://github.com/NetApp/terraform-provider-netapp-cloudmanager/blob/main/examples/gcp/resources.tf#L89

**Comment by chuyich** [comment]
Would you please run with debug flag and share the log with us?
**TF_LOG="DEBUG" terraform apply**

**Comment by pablogarciaarevalo** [comment]
Sure, Here you have. [cvo-ha-zone-issue.txt](https://github.com/NetApp/terraform-provider-netapp-cloudmanager/files/10350365/cvo-ha-zone-issue.txt)
Thank you

**Comment by chuyich** [comment]
For the value of zone, it should be "europe-southwest1-a"

**Comment by pablogarciaarevalo** [comment]
Yes. But this is the reason that I opened this issue. This parameter makes sense for a CVO Single Node deployment. But it should not required in a CVO High Availability deployment, the parameters 'node1_zone', 'node2_zone', and 'mediator_zone' should be the good ones.

**Comment by chuyich** [comment]
Thanks for pointing this. Raised DEVOPS-5814 to track this issue.

**Comment by lonico** [comment]
fixed in VERSION 23.1.1
